src/ui-ace.js

```diff
@@ -224,11 +224,13 @@ angular.module('ui.ace', [])
             return function (e) {
               var newValue = session.getValue();
 
-              if (ngModel && newValue !== ngModel.$viewValue &&
+              if (ngModel && //newValue !== ngModel.$viewValue &&
```

Comment:
Let's remove the dead code. Keeping it here doesn't help us and that's what Git is supposed to handle for us.